src/main/java/dev/ai4j/openai4j/GenericHeaderInjector.java

```diff
@@ -0,0 +1,29 @@
+package dev.ai4j.openai4j;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import okhttp3.Interceptor;
+import okhttp3.Request.Builder;
+import okhttp3.Response;
+
+class GenericHeaderInjector implements Interceptor {
+	private final Map<String, String> headers = new HashMap<>();
```

Comment:
I'm curious about why indentation is 8?? 😄  Default in project here is 4.
Find the fresh `.prettierrc` file in the project root - the config for the formatting tool `Prettier`.

Reply:
No idea? My IDE is set up as indent by 2, so no idea where 8 came from....

Reply:
It looks like it is indented by 4 to me, regardless of what editor I open it in...

![image](https://github.com/ai-for-java/openai4j/assets/363447/7b51d4e5-acd6-4eaf-9857-30deafb6d37f)

Reply:
looks like there are tabs, just format with 4 spaces, that's it 😄

Reply:
How's that?

Reply:
There must be related setting in every IDE for that. I'm already approving then.

Reply:
now it's fine!! 👍